native/build_test.go

```diff
@@ -47,10 +46,10 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 	it.Before(func() {
 		var err error
 
-		ctx.Application.Path, err = ioutil.TempDir("", "build-application")
+		ctx.Application.Path, err = os.MkdirTemp("", "build-application")
```

Comment:
Thank you for fixing up these deprecated `ioutil` calls. We are not doing this in a lot of places yet, but I've been trying to update tests that create temp directories to use `t.TempDir()` instead, https://pkg.go.dev/testing#T.TempDir. It simplifies things because the cleanup happens automatically.

Unless there's a need to have a greater degree of control over what happens with the creation/cleanup of the temp directory.

Reply:
Nevermind, I'll do the change!

Reply:
done!

Reply:
Thanks!